tests/test_chatapproach.py

```diff
@@ -5,7 +5,7 @@
 
 def test_get_search_query():
     chat_approach = ChatReadRetrieveReadApproach(
-        None, "", "gpt-35-turbo", "gpt-35-turbo", "", "", "", "", "en-us", "lexicon"
+        None, None, "", "gpt-35-turbo", "gpt-35-turbo", "", "", "", "", "en-us", "lexicon"
```

Comment:
At this point, you could make this a module-level fixture, as I think we reuse it a bunch of times. No big deal but have thought about that in previous PRs to this file.